internal/node/node_loader.js

```diff
@@ -84,18 +88,48 @@ function resolveToModuleRoot(path) {
  */
 function loadRunfilesManifest(manifestPath) {
   if (DEBUG) console.error(`node_loader: using manifest ${manifestPath}`);
+
+  // Create the manifest and reverse manifest maps.
   const runfilesManifest = Object.create(null);
   const reverseRunfilesManifest = Object.create(null);
   const input = fs.readFileSync(manifestPath, {encoding: 'utf-8'});
+  let workspaceRoot;
   for (const line of input.split('\n')) {
     if (!line) continue;
     const [runfilesPath, realPath] = line.split(' ');
     runfilesManifest[runfilesPath] = realPath;
     reverseRunfilesManifest[realPath] = runfilesPath;
+
+    // Determine workspace root to convert absolute paths into runfile paths.
+    // This only works if there is at least one runfile in the workspace root, but that is
+    // also the only case when we need to map back to the runfiles.
```

Comment:
include a pointer to the bug you filed about how Bazel ought to do this

Reply:
Done